docs/UsersGuide/source/LAMGrids.rst

```diff
@@ -87,7 +87,7 @@ Predefined 13-km Grid
 
    *The boundary of the RRFS_CONUS_13km computational grid (red) and corresponding write-component grid (blue).*
 
-The ``RRFS_CONUS_13km`` grid (:numref:`Fig. %s <RRFS_CONUS_13km>`) covers the full :term:`CONUS`. This grid is meant to be run with the ``FV3_GFS_v16`` physics suite. The ``FV3_GFS_v16`` physics suite uses convective :term:`parameterizations`, whereas the other supported suites do not. Convective parameterizations are necessary for low-resolution grids because convection occurs on scales smaller than 25km and 13km. 
+The ``RRFS_CONUS_13km`` grid (:numref:`Fig. %s <RRFS_CONUS_13km>`) covers the full :term:`CONUS`. This grid is meant to be run with the ``FV3_GFS_v16`` physics suite. The ``FV3_GFS_v16`` physics suite uses convective :term:`parameterizations`, whereas the other supported suites do not. Convective parameterizations are necessary for low-resolution grids because convection occurs on scales smaller than 25-km and 13-km. 
 
```

Comment:
Either's fine, but since we are using dashes everywhere else (e.g., 25-km), It should be 25- and 13-km if I change it. Doesn't seem necessary, so I'm going to leave it.